Bio/Nexus/Nexus.py

```diff
@@ -934,7 +934,7 @@ def _statelabels(self, options):
         pass
 
     def _matrix(self, options):
-        """ Creates a matrix for NEXUS object."""
+        """Creates a matrix for NEXUS object."""
```

Comment:
This is a private method (starts with an underscore, not one of the underscore-underscore special methods like ``__len__``), so could you add ``... (PRIVATE)."""``` to the end of the line:

```python
   """Creates a matrix for NEXUS object (PRIVATE)."""
```